src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -173,12 +170,60 @@ impl<'value, 'top> BinaryValueWriter_1_1<'value, 'top> {
         todo!()
     }
 
-    pub fn write_string<A: AsRef<str>>(self, _value: A) -> IonResult<()> {
-        todo!()
+    pub fn write_string<A: AsRef<str>>(mut self, value: A) -> IonResult<()> {
+        const STRING_OPCODE: u8 = 0x80;
+        const STRING_FLEX_UINT_LEN_OPCODE: u8 = 0xF8;
+        self.write_text(STRING_OPCODE, STRING_FLEX_UINT_LEN_OPCODE, value.as_ref())
     }
 
-    pub fn write_symbol<A: AsRawSymbolTokenRef>(self, _value: A) -> IonResult<()> {
-        todo!()
+    pub fn write_symbol<A: AsRawSymbolTokenRef>(mut self, value: A) -> IonResult<()> {
+        const SYMBOL_OPCODE: u8 = 0x90;
+        const SYMBOL_FLEX_UINT_LEN_OPCODE: u8 = 0xF9;
+        match value.as_raw_symbol_token_ref() {
+            RawSymbolTokenRef::SymbolId(sid) => self.write_symbol_id(sid),
+            RawSymbolTokenRef::Text(text) => {
+                self.write_text(SYMBOL_OPCODE, SYMBOL_FLEX_UINT_LEN_OPCODE, text.as_ref())
+            }
+        }
+    }
+
+    #[inline]
+    fn write_symbol_id(&mut self, symbol_id: SymbolId) -> IonResult<()> {
+        match symbol_id {
+            0..=255 => {
+                self.push_byte(0xE1); // Single-byte FixedUInt follows
+                FixedUInt::write_u64(self.encoding_buffer, symbol_id as u64)?;
```

Comment:
This would also work, right?
```suggestion
                self.push_bytes([0xE1, symbol_id as u8]);
```

Reply:
Yep, nice!